nixos/modules/services/printing/cupsd.nix

```diff
@@ -71,7 +71,7 @@ let
     };
 
   cupsFilesFile = writeConf "cups-files.conf" ''
-    SystemGroup root wheel
+    SystemGroup root wheel ${lib.optionalString (cfg.extraSystemGroup != null) cfg.extraSystemGroup}
```

Comment:
`man cups-files.conf` says the default is `"admin", "lpadmin", "root", "sys", and/or "system"`. Why don't we at least add "lpadmin" unconditionally, update the documentation, and not add the extraSystemGroup option?

Reply:
IMHO, no. Whether or not to have an option (even read-only) is an orthogonal issue. The module could default to create and use an "lpadmin" group. And having asserts for groups doesn't require having options for each group either.